ansible/vars/epoch/main.yml

```diff
@@ -0,0 +1,44 @@
+api_base_uri: http://{{ public_ipv4 }}:{{ epoch_config.http.external.port }}/v2
+configure_peers: false
+seed_peers:
+  - "35.177.201.208"
```

Comment:
I assume this environment, and addresses must be documented in doc of epoch repo?